.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,34 @@
+name: 🐛 Bug report
+description: Create a report to help us improve
+labels: bug
+title: "[Bug]: "
```

Comment:
I know you haven't changed this, but reviewing this again has me wondering if we should remove the "[Bug]:" and "[Feature Request]:" from the title.

I think the "bug" and "enhancement" labels do this categorization already, so we don't need to encode the information twice.  If you agree, I think we should make this change in both repositories.

Reply:
Not a hill I'm willing to die on, but I also don't feel like it degrades things much either way. I feel like the title is easier to scan, and the labels are more for filtering later on so they server different purposes in my mind. Happy to go either way?

Reply:
If you could go either way, I think I'd prefer to remove it.  It leaves more room (horizontally) for other characters in the title, and I do the filtering by bug/enhancement anyway before scanning the list.

Reply:
Done in 8c5c1b1 and c6ee22d. Will create an issue on P3